Assets/VRTK/Prefabs/Resources/Sprites.meta

```diff
@@ -0,0 +1,9 @@
+fileFormatVersion: 2
```

Comment:
There is already a directory called `Images` would it be better off just putting this in there rather than another directory called `Sprites`?